src/main/resources/templates/repo/kinesis-log-streams.json

```diff
@@ -66,17 +148,42 @@
 		"backupDisabled": true,
 		"tableDescriptor": {
 			"name": "fileHandleAssociationsRecords",
-			"columns": {
-				"timestamp": "timestamp",
-				"associateType": "string",
-				"associateId": "bigint",
-				"fileHandleId": "bigint",
-				"stack": "string",
-				"instance": "string"
+			"description" : "The table contains file handle association records. A FileHandleAssociation record is a FileHandle (identified by its id) along with a Synapse object (identified by its id and type).",
+			"columns": [{
+				"name": "timestamp",
+				"type" : "timestamp",
+				"comment" : "The time when the association was data was collected."
 			},
-			"partitionKeys": {
-				"dt": "string"
+			{
+				"name": "associateType",
+				"type" : "string",
+				"comment" : "The type of the Synapse object that wraps the file."
 			},
+			{
+				"name": "associateId",
+				"type" : "string",

```

Comment:
This is the wrong datatype, should be bigint (see previous state).

Reply:
Make sure to run a full build of the services on top of this stack.

Reply:
I didn't check all the changes, but the datatypes should be preserved unless there is a reason to change them, the code repository code should reflect any change.